int8 NORTH=1
int8 SOUTH=2
int8 EAST=3
int8 WEST=4
int8 orientation
bool isBig
int8 number
---